finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java

```diff
@@ -58,13 +62,21 @@ public Response getSingleOrder(@PathParam("orderId") String orderId){
     @GET
     @Path("/table/{tableId}")
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "listOrdersByTable",
+               summary = "List the orders for a table",
+               description = "This operation retrieves all orders " +
+                   "of the provided tableId from the Status service.")
     @Tag(name = "Status")
     public Response getOrdersList(@PathParam("tableId") String tableId){
         return statusClient.getOrdersList(tableId);
     }
     
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "resetOrder",
+               summary = "Clear all orders",
+               description = "This operation remove all orders " + 
+                   "in the Status service.")
```

Comment:
Update the description to
"This operation removes all orders in the Status service."

Reply:
updated